pkg/operator/starter.go

```diff
@@ -81,6 +90,25 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 	if err != nil {
 		return err
 	}
+	staticResourceController := staticresourcecontroller.NewStaticResourceController(
+		"OpenshiftAPIServerStaticResources",
+		v311_00_assets.Asset,
+		[]string{
+			"v3.11.0/openshift-apiserver/ns.yaml",
+			"v3.11.0/openshift-apiserver/apiserver-clusterrolebinding.yaml",
+			"v3.11.0/openshift-apiserver/svc.yaml",
+			"v3.11.0/openshift-apiserver/sa.yaml",
+			"v3.11.0/openshift-apiserver/trusted_ca_cm.yaml",
+		},
+		(&resourceapply.ClientHolder{}).WithKubernetes(kubeClient).WithAPIExtensionsClient(apiExtensionsClient),
```

Comment:
it was always like that or have they changed something in go 1.13? I think that taking the address of an anonymous obj didn't work.

Reply:
> it was always like that or have they changed something in go 1.13? I think that taking the address of an anonymous obj didn't work.

Uh, I just added this.  It seemed to compile